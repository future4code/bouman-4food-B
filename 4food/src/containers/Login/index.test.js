import React from "react";
import { LoginPage } from "./index";
import renderer from "react-test-renderer";

describe("Test Login", () =>{
    test("Snapshot Login", () => {
        const testLogin = renderer.create(<LoginPage goToSignUp={()=>{}} />)
        .toJSON();
        expect(testLogin).toMatchSnapshot();
    });

});