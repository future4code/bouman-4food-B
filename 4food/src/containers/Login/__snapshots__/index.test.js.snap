// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Login Snapshot Login 1`] = `
<div
  className="sc-kGXeez gWdMNk"
>
  <div>
    <img
      className="sc-kpOJdX gcLKDy"
      src="logo-vermelho.svg"
    />
  </div>
  <h4>
    Entrar
  </h4>
  <form
    className="sc-dxgOiQ bGXuvX"
    onSubmit={[Function]}
  >
    <div
      className="MuiFormControl-root-1"
    >
      <label
        className="MuiFormLabel-root-16 MuiFormLabel-required-21 MuiInputLabel-required-9 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13 MuiInputLabel-shrink-12 MuiInputLabel-outlined-15"
        data-shrink={true}
      >
        E-mail
        <span
          className="MuiFormLabel-asterisk-22"
        >
           *
        </span>
      </label>
      <div
        className="MuiInputBase-root-36 MuiOutlinedInput-root-23 MuiInputBase-formControl-37"
        onClick={[Function]}
      >
        <fieldset
          aria-hidden={true}
          className="MuiPrivateNotchedOutline-root-53 MuiOutlinedInput-notchedOutline-30"
          style={
            Object {
              "paddingLeft": 8,
            }
          }
        >
          <legend
            className="MuiPrivateNotchedOutline-legend-54"
            style={
              Object {
                "width": 0,
              }
            }
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </legend>
        </fieldset>
        <input
          aria-invalid={false}
          className="MuiInputBase-input-46 MuiOutlinedInput-input-31 MuiInputBase-inputType-49"
          disabled={false}
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="email@email.com"
          required="true"
          type="email"
          value=""
        />
      </div>
    </div>
    <br />
    <div
      className="MuiFormControl-root-1"
      pattern=".{6,}"
    >
      <label
        className="MuiFormLabel-root-16 MuiFormLabel-required-21 MuiInputLabel-required-9 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13 MuiInputLabel-shrink-12 MuiInputLabel-outlined-15"
        data-shrink={true}
      >
        Senha
        <span
          className="MuiFormLabel-asterisk-22"
        >
           *
        </span>
      </label>
      <div
        className="MuiInputBase-root-36 MuiOutlinedInput-root-23 MuiInputBase-formControl-37 MuiInputBase-adornedEnd-41 MuiOutlinedInput-adornedEnd-27"
        onClick={[Function]}
      >
        <fieldset
          aria-hidden={true}
          className="MuiPrivateNotchedOutline-root-53 MuiOutlinedInput-notchedOutline-30"
          style={
            Object {
              "paddingLeft": 8,
            }
          }
        >
          <legend
            className="MuiPrivateNotchedOutline-legend-54"
            style={
              Object {
                "width": 0,
              }
            }
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </legend>
        </fieldset>
        <input
          aria-invalid={false}
          className="MuiInputBase-input-46 MuiOutlinedInput-input-31 MuiInputBase-inputType-49 MuiInputBase-inputAdornedEnd-52 MuiOutlinedInput-inputAdornedEnd-35"
          disabled={false}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="Mínimo 6 caracteres"
          required="true"
          type="password"
          value=""
        />
        <div
          className="MuiInputAdornment-root-55 MuiInputAdornment-positionEnd-58"
        >
          <button
            aria-label="Toggle password visibility"
            className="MuiButtonBase-root-66 MuiIconButton-root-60"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex="0"
            type="button"
          >
            <span
              className="MuiIconButton-label-65"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root-69"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M0 0h24v24H0zm0 0h24v24H0zm0 0h24v24H0zm0 0h24v24H0z"
                  fill="none"
                />
                <path
                  d="M12 7c2.76 0 5 2.24 5 5 0 .65-.13 1.26-.36 1.83l2.92 2.92c1.51-1.26 2.7-2.89 3.43-4.75-1.73-4.39-6-7.5-11-7.5-1.4 0-2.74.25-3.98.7l2.16 2.16C10.74 7.13 11.35 7 12 7zM2 4.27l2.28 2.28.46.46C3.08 8.3 1.78 10.02 1 12c1.73 4.39 6 7.5 11 7.5 1.55 0 3.03-.3 4.38-.84l.42.42L19.73 22 21 20.73 3.27 3 2 4.27zM7.53 9.8l1.55 1.55c-.05.21-.08.43-.08.65 0 1.66 1.34 3 3 3 .22 0 .44-.03.65-.08l1.55 1.55c-.67.33-1.41.53-2.2.53-2.76 0-5-2.24-5-5 0-.79.2-1.53.53-2.2zm4.31-.78l3.15 3.15.02-.16c0-1.66-1.34-3-3-3l-.17.01z"
                />
              </svg>
            </span>
            <span
              className="MuiTouchRipple-root-104"
            />
          </button>
        </div>
      </div>
    </div>
    <br />
    <button
      className="MuiButtonBase-root-66 MuiButton-root-78 MuiButton-contained-89 MuiButton-containedPrimary-90 MuiButton-raised-92 MuiButton-raisedPrimary-93"
      disabled={false}
      onBlur={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="submit"
    >
      <span
        className="MuiButton-label-79"
      >
        Entrar
      </span>
      <span
        className="MuiTouchRipple-root-104"
      />
    </button>
  </form>
  <p>
    Não possui cadastro? 
    <button
      className="MuiButtonBase-root-66 MuiButton-root-78 MuiButton-text-80 MuiButton-flat-83 MuiButton-sizeSmall-101"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="button"
    >
      <span
        className="MuiButton-label-79"
      >
         
        <h5>
          Clique aqui.
        </h5>
      </span>
      <span
        className="MuiTouchRipple-root-104"
      />
    </button>
     
  </p>
</div>
`;
